src/main/java/AutomationPracticeLyrics/LostOnYouLyricsPage.java

```diff
@@ -0,0 +1,71 @@
+package AutomationPracticeLyrics;
+
+import Utils.WaitMethods;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.*;
+
+public class LostOnYouLyricsPage {
+    @FindBy(xpath = "//pre[@id = 'lyric-body-text']")
```

Comment:
From now on, when you have id as selector, please use "id" as selector not xpath.
The uniqueness of id makes things way more convenient.